catalog/imaging/catalog-imaging-transformer/src/main/java/com/connexta/alliance/transformer/nitf/NitfInputTransformer.java

```diff
@@ -59,7 +59,7 @@
 
     private static final String ID = "ddf/catalog/transformer/nitf";
 
-    private static final String MIME_TYPE = "image/nitf";
+    static final String MIME_TYPE = "image/nitf";
```

Comment:
An [`ExceptionInInitializerError`](http://docs.oracle.com/javase/8/docs/api/java/lang/ExceptionInInitializerError.html) would be appropriate.